src/Service/Context/PageContext.php

```diff
@@ -182,6 +184,19 @@ private function setPageContextCredential($credential_settings) {
   }
 
   /**
+   * Set any Lift configuration values to the page context.
+   *
+   * @param $settings \Drupal\Core\Config\ImmutableConfig
+   *   The lift settings values.
+   */
+  private function setPageContextConfig($settings) {
+    $replacement_mode_setting = $settings->get('content_replacement_mode');
+    if (SettingsHelper::isValidContentReplacementMode($replacement_mode_setting)) {
+      $this->pageContext['contentReplacementMode'] = $replacement_mode_setting;
+    }
+  }
```

Comment:
If we make the changes as recommended above, here we would need:

`$content_settings = $settings->get('content');`
and
`$this->setPageContextContent($content_settings);`